packages/contracts/test/cyberMarketplace.spec.ts

```diff
@@ -0,0 +1,210 @@
+import chai from 'chai'
+import { solidity } from 'ethereum-waffle'
+import { ethers, network } from 'hardhat'
+import { expect } from 'chai'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { BigNumber, Contract } from 'ethers'
+
+import type {
+  DummyContract,
+  DummyNFT,
+  CyberMarketplace,
+} from '../typechain-types'
+
+const contracts: {
+  marketplace: CyberMarketplace
+  nft: DummyNFT
+  dummy: DummyContract
+} = {} as any
+
+const signers: Record<string, SignerWithAddress> = {}
+
+const KONG = 1
+const PUDGY = 2
+
+function days(n: number) {
+  return n * 24 * 60 * 60
+}
+
+function eth(n: number) {
+  return BigNumber.from(n).mul(BigNumber.from(10).pow(18))
+}
+
+function big(n: number) {
+  return BigNumber.from(n)
+}
+
+function delay(ms: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms)
+  })
+}
+
+describe('CyberMarketplace', function () {
+  beforeEach(async () => {
+    let signerArr = await ethers.getSigners()
```

Comment:
use const in place of let when variable are immutable, it's the case for most of them in this file

Reply:
Used to do this (if you check my twitter feed and blog you'll see I'm FP purist). 

Point is since we're in a mutable world (JS) when you want to change the code to reassign a variable you search back the declaration site and change `const` to `let`. Happened to me so much (esp. in tests) that I started to declare vars with `let` by default.

I guess it's not the right place for the discussion, but IMO `const` provides little benefit in JS since you can mutate values and also shadow previously declared ones.

Reply:
I will be happy to discuss later, please use const when the value is immutable.
Tips: don't be afraid in the packages you wouldn't need to re-assign it later, believe me